onadata/apps/api/viewsets/user_profile_viewset.py

```diff
@@ -142,6 +144,29 @@ def change_password_attempts(request):
     return 1
 
 
+def invalidate_and_regen_tokens(user):
```

Comment:
Can we have this util function in `onadata.libs.authentication` module rather than a view set? Especially since it is also used by the `CustomPasswordResetTokenGenerator` class. I think it clutters the viewset.

Reply:
Yes, I'll make that change.